lib/shipping_agent/github/notification.rb

```diff
@@ -20,6 +21,34 @@ def update(status, description, deploy)
 
       private
 
+      # Github has an option to do this automaticly, but it only works
+      # for non-production environments.
+      def mark_old_deployments_inactive(deploy)
+        active_old_deployments(deploy).each { |deployment| mark_inactive(deployment) }
+      end
+
+      def active_old_deployments(current)
+        old_deployments(current).reject do |deployment|
+          current.url == deployment.url ||
+            deployment.rels[:statuses].get.data.any? { |s| s.state == "inactive" }
```

Comment:
`mark_old_deployments_inactive` is looking at all of the existing Github deployments to the current environment and adding a new status with a state of inactive to any deployment that does not have it.  This is what makes the deploy thingy in Github get greyed out.

This stuff is really a Github concern, these are Github deployments.  Not our domain model Deploy or a k8s deployment.